resolver-dns/src/test/java/io/netty/resolver/dns/SearchDomainTest.java

```diff
@@ -292,9 +290,8 @@ public void testExceptionMsgContainsSearchDomain() throws Exception {
         assertTrue(fut.await(10, TimeUnit.SECONDS));
         assertFalse(fut.isSuccess());
         final Throwable cause = fut.cause();
-        assertThat(cause, instanceOf(UnknownHostException.class));
-        assertThat("search domain is included in UnknownHostException", cause.getMessage(),
-            containsString("foo.com"));
+        assertInstanceOf(UnknownHostException.class, cause);
+        assertThat(cause.getMessage()).contains("foo.com");
```

Comment:
If you want to preserve the exception description, you can include it with the `as()` method of the asserter returned from `assertThat()`.

For instance:

```java
assertThat(cause.getMessage()).as("search domain is included in UnknownHostException").contains("foo.com");
```